src/inverseRobot.js

```diff
@@ -24,7 +24,36 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.keys(robot);
+  const keys = Object.values(robot);
+  const repairStats = {};
+
+  for (let j = 0; j < values.length; j++) {
+    let sumk = 0;
```

Comment:
it is not understandable what do the variables mean: sumk, sumv, ch. Use more understandable names